webapp/tests/test_finders.py

```diff
@@ -153,6 +158,32 @@ def listdir_mock(d):
             os.listdir = self._original_listdir
             self.wipe_whisper()
 
+    def test_standard_finder_gzipped_whisper(self):
+        def listdir_mock(d):
+            self._listdir_counter += 1
+            return self._original_listdir(d)
+
```

Comment:
This was how it was done in the other similar test.  Other than that, I don't know of a reason off hand to choose this way over `mock.patch()`.

I think we're pretty inconsistent with how we mock things in the tests.  It may be useful to define some standards ¯\\_(ツ)_/¯